ProjectSidewalk/SidewalkWebpage
# #2840: Updated gallery cards for label types that do not support severity ratings

Resolves #2840

Changed Gallery cards with unsupported severity ratings to have grayed out severity ratings, as well as a tooltip that, wehn hovered over, explains why they have no rating.

##### Before/After screenshots (if applicable)
Before:
![image](https://user-images.githubusercontent.com/77756028/176738015-d7ab975f-7626-49df-a3c2-dc219e886eb5.png)

After:
![image](https://user-images.githubusercontent.com/77756028/176737894-5fffb02a-4c10-48e9-81f0-3e9350eb391d.png)

##### Testing instructions
1. Go to the gallery tab
2. Under the "Show" drop down on the left, hit "Pedestrian Signal," or any label type that does not support severity ratings. (if the server you're working on doesn't have any pedestrian signals tagged, go into the audit tab and throw in a couple random pedestrian signal labels, not in production server ofc!!!)
3. Check that the Severity Rating is grayed out on all cards, and that a tooltip is there. When hovered over, a message box should appear.
4. Check in all languages!

##### Things to check before submitting the PR
